### DISCUSSION TITLE
Not able to signup after Kutt docker setup

### DISCUSSION THREAD

**Discussion by Biswadeepgiri** [thedevs-network/kutt#755] [General]
After setting up kutt in my local machine using docker when I am trying to signup using a new email and password then it is showing this ->

POST http://localhost:3000/api/v2/auth/signup 500 (Internal Server Error)

How can I fix it ?

**Comment by poeti8** [comment]
Could be because your email server is not configured or is not getting connected. Same for Postgres or Redis.

**Comment by Biswadeepgiri** [comment]
I configured the email server and configured Postgres and Redis too but the issue still persists.
